Adyen/UI/Form/FormView.swift

```diff
@@ -37,21 +44,25 @@ internal final class FormView: UIScrollView {
         stackView.addArrangedSubview(itemView)
     }
 
-    // MARK: - Stack View
-    
-    private lazy var stackView: UIStackView = {
-        let stackView = UIStackView()
-        stackView.axis = .vertical
-        stackView.alignment = .fill
-        stackView.preservesSuperviewLayoutMargins = true
-        stackView.translatesAutoresizingMaskIntoConstraints = false
-        return stackView
-    }()
+    // MARK: - Private
 
-    // MARK: - Layout
+    private func setup() {
+        preservesSuperviewLayoutMargins = true
+
+        addSubviews()
+        setupLayout()
+    }
+
+    private func addSubviews() {
+        addSubview(stackView)
+    }
 
-    private func configureConstraints() {
-        stackView.adyen.anchor(inside: self)
-        stackView.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
+    private func setupLayout() {
+        NSLayoutConstraint.activate([
```

Comment:
This is the same as it was before, right?
`stackView.adyen.anchor(inside: self)`

Reply:
Yes, refactoring to use the helper method

Reply:
I think it's different just because of the last line `lessThanOrEqualTo`